codegen/lib/coin_test_gen.rb

```diff
@@ -61,7 +61,7 @@ def generate_coin_test_file(coin, templateFile, overwriteExisting = true)
     template = ERB.new(File.read(path), nil, '-')
     result = template.result(binding)
 
-    folder = 'tests/' + format_name(coin['name'])
+    folder = 'tests/' + format_name(coin['id'])
```

Comment:
Let's avoid making hundreds unnecessary file changes, even you do a upper case of `id` here you can avoid so many changes

Reply:
Ok, I agree. I changed the approach.